BlueprintUI/Sources/Environment/Keys/AccessibilityLinkKey.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+
+extension Environment {
+    private enum AccesibilityLinkKey: EnvironmentKey {
+        static var defaultValue: String? {
+            UIImage(systemName: "link")?.accessibilityLabel
+        }
+    }
+
+    /// The localised accessibility label elements should use when handling links.
+    ///
+    /// Defaults to `UIImage(systemName: "link")?.accessibilityLabel`.
+    public var linkAccessibility: String? {
```

Comment:
It's used as both an `accessibilityLabel` and `accessibilityValue` in different places (inserted into the `accessibilityLabel` for the entire textLabel vs the 'accessibilityValue' on the link element itself) so I wanted to avoid using one or the other explicitly in the naming. 
I thought about `localizedLinkAccessibilityString` but thats a mouthful and imbedding the type in the name seemed like my Objective C accent leaking through.